app/assets/javascripts/admin/application.js

```diff
@@ -16,6 +16,16 @@
 
 $(document).on('turbolinks:load', function() {
 
+  if ($('#age_report').length) {
```

Comment:
I can't see that folder inside `javascripts/admin`. There's a `javascripts/gobierto_budget_consultations` folder, but it has the Vue app.